docs/installation/helm.rst

```diff
@@ -1,6 +1,37 @@
 .. _installation_helm:
 
-Helm (TODO)
-===========
+Helm
+====
 
-TODO
+Deploying to Kubernetes is possible with Helm_. The GPP-Woo maintainers publish
+`charts <https://github.com/GPP-Woo/charts>`_ for the full stack (umbrella chart) and
+the invidual components. You can use the former as-is, or use it as inspiration to build
+your own umbrella chart.
+
+The remainder of the documentation assumes you're deploying the full stack.
+
+Add the repositories
+--------------------
+
+To deploy the charts, first add the necessary repositories:
+
+.. code-block:: bash
+
+    helm repo add gpp-stack https://GPP-Woo.github.io/charts
+
+Deploy the stack
+----------------
+
+To deploy the stack, ensure you have your own ``values.yml`` file overriding the relevant
+settings/defaults from the chart itself - see
+`GitHub <https://github.com/GPP-Woo/charts/tree/main/charts/GPP-stack>`_ for a reference.
+
+.. code-block:: bash
+
+    helm repo update
+    helm install \
+        -f myvalues.yml \
+        my-gpp-woo \
+        gpp-stack
```

Comment:
Given the command above, I think this should be:
```
helm install -f values.yaml gpp-stack gpp-stack/gpp-stack
```
(After `helm repo add gpp-stack https://GPP-Woo.github.io/charts` locally, if I see which repositories I have available with `helm search repo`, the umbrella chart is listed as `gpp-stack/gpp-stack`)

Reply:
updated!